Makefile

```diff
@@ -10,14 +10,18 @@ CWD = $(shell pwd)
 PYTHON = $(shell if python --version 2>&1 | egrep -q 'Python 3\..*' ; then echo "python"; else echo "python3"; fi)
```

Comment:
why not always `python3`?

Reply:
Can I assume that all systems have Python 3 as python3? I believe I've encountered cases where this was no true, but I'm not entirely sure.

Reply:
Oh, I agree with that - but I would expect an install in a virtualenv or conda env to be preferred.  At the end it is hard to map the gazillion of python deployment options into a script :-/

Reply:
Shouldn't we use the more general solution?

Reply:
I apologize - that comment was not meant for this thread, I somehow screwed up...

> Can I assume that all systems have Python 3 as python3?
Yes, I believe that this is the case, I did not see that not being the case anywhere (we use a fixed `#!/usr/bin/env python3` shebang on all our scripts).